data/IL/incentives.json

```diff
@@ -0,0 +1,1110 @@
+    "end_date": 2024
+  },
+  {
+    "id": "IL-45",
+    "authority_type": "utility",
+    "authority": "il-mid-american-energy",
+    "payment_methods": [
+      "pos_rebate"
+    ],
+    "item": "heat_pump_air_conditioner_heater",
+    "program": "il_midAmericanEnergyCompany_residentialInstantRebates",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 375,
+      "maximum": 375
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "Up to $375 point-of-sale rebate for eligible air source heat pumps, depending on efficiency."
+    },
+    "start_date": 2024,
+    "end_date": 2024
+  },
+  {
+    "id": "IL-46",
+    "authority_type": "utility",
+    "authority": "il-mid-american-energy",
+    "payment_methods": [
+      "pos_rebate"
+    ],
+    "item": "heat_pump_air_conditioner_heater",
+    "program": "il_midAmericanEnergyCompany_residentialInstantRebates",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 300,
+      "maximum": 300
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "Up to $300 point-of-sale rebate for eligible ductless mini-split heat pumps."
+    },
+    "start_date": 2024,
+    "end_date": 2024
+  },
+  {
+    "id": "IL-47",
+    "authority_type": "utility",
+    "authority": "il-mid-american-energy",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "smart_thermostat",
+    "program": "il_midAmericanEnergyCompany_residentialInstantRebates",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 75,
+      "maximum": 75
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "Up to $100 rebate for ENERGY STAR® Smart Thermostats, up to 70% of equipment cost."
+    },
+    "start_date": 2024,
+    "end_date": 2024
+  },
+  {
+    "id": "IL-48",
+    "authority_type": "utility",
+    "authority": "il-mid-american-energy",
+    "payment_methods": [
+      "pos_rebate"
+    ],
+    "item": "geothermal_heating_installation",
+    "program": "il_midAmericanEnergyCompany_residentialInstantRebates",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 1200,
+      "maximum": 1200
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "Up to $1200 rebate for eligible geothermal heat pumps, depending on efficiency."
+    },
+    "start_date": 2024,
+    "end_date": 2024
+  },
+  {
+    "id": "IL-49",
+    "authority_type": "utility",
+    "authority": "il-mid-american-energy",
+    "payment_methods": [
+      "pos_rebate"
+    ],
+    "item": "heat_pump_water_heater",
+    "program": "il_midAmericanEnergyCompany_residentialInstantRebates",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 225,
+      "maximum": 225
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "Up to $225 per unit."
```

Comment:
Definitely too short – needs to mention the kind of appliance.

Reply:
updated